Makefile

```diff
@@ -86,51 +90,82 @@ integration-test-full: integration-env-down integration-env-up integration-test
 
 integration-env-up-ci: emily-cdk-synth
 	docker compose --file docker/docker-compose.ci.yml up --detach --quiet-pull
-	@echo "Wait for aws resources to be set up..."
-	@while docker compose --file docker/docker-compose.ci.yml ps | grep -q 'emily-aws-setup'; do echo "waiting..." && sleep 1; done
+	DYNAMODB_ENDPOINT=http://localhost:8000 \
+		INPUT_CDK_TEMPLATE=./emily/cdk/cdk.out/EmilyStack.template.json \
+		OUTPUT_CDK_TEMPLATE=./emily/cdk/cdk.out/EmilyStack.devenv.template.json \
+		LOCAL_LAMBDA_PATH=empty.zip \
+		TRUSTED_REORG_API_KEY=testApiKey \
+		python3 docker/sbtc/emily-aws-setup/initialize.py
+	cargo build --bin emily-server
 	AWS_ACCESS_KEY_ID=xxxxxxxxxxxx \
-	AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxx \
-	AWS_REGION=us-west-2 \
-	TRUSTED_REORG_API_KEY=testApiKey \
-	cargo run --bin emily-server -- \
-		--host 127.0.0.1 --port 3031 --dynamodb-endpoint http://localhost:8000 > ./target/emily-server.log 2>&1 &
+		AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxx \
+		AWS_REGION=us-west-2 \
+		TRUSTED_REORG_API_KEY=testApiKey \
+		cargo run --bin emily-server -- \
+			--host 127.0.0.1 --port 3031 --dynamodb-endpoint http://localhost:8000 > ./target/emily-server.log 2>&1 &
 
 integration-env-down-ci:
 	docker compose --file docker/docker-compose.ci.yml down
 	@echo "killing emily server process..."
 	ps -ef | awk  '/[e]mily-server/{print $$2}' | xargs kill -9
 
-.PHONY: integration-env-up integration-test integration-env-up integration-test-full
+.PHONY: integration-env-up integration-test integration-test-build integration-env-up integration-test-full
 
-# Emily API
-# ----------------------------------------------------
+# ##############################################################################
+# DEVENV (development testing environment)
+# ##############################################################################
 
-# Project Names
-## Cargo crates
-EMILY_HANDLER_PROJECT_NAME := emily-handler
-EMILY_OPENAPI_SPEC_PROJECT_NAME := emily-openapi-spec
-## Node projects
-EMILY_CDK_PROJECT_NAME := emily-cdk
+devenv-no-sbtc-up:
+	docker compose -f docker/docker-compose.yml --profile default --profile bitcoin-mempool up
```

Comment:
Can we add `-d` here (and maybe also in `devenv-sbtc-up`)? To align with `devenv-up` (and avoid having logs spammed to the console).

Even better if we could supply some extra flags when invoking the targets in this section (eg, being able to `make devenv-down -- -v -t 0`, not sure if it's something we can do with make)

Reply:
[816e2b3](https://github.com/stacks-network/sbtc/pull/1238/commits/816e2b37e0cf5209a25cd5c50313c911d058dc33)
[f8327b8](https://github.com/stacks-network/sbtc/pull/1238/commits/f8327b8d23c94156aa0c8c73050c12f7c7a6536e)
[cb4947b](https://github.com/stacks-network/sbtc/pull/1238/commits/cb4947bb4397208f4f93ef9fd26d1ef4a1b396af)

Yeah `make` doesn't support arguments like that... have to pass it like `make xxx ARGS="-t 0 -v"` etc.